src/ConcreteOperations/convex_hull.jl

```diff
@@ -457,8 +462,9 @@ function monotone_chain!(points::Vector{VN}; sort::Bool=true) where {N,VN<:Abstr
     end
 
     if sort
+        # _zero_abs is necessary because floating-point arithmetic distinguishes between 0.0 and -0.0, which leads to incorrect sorting
```

Comment:
```suggestion
        # _zero_abs is necessary because floating-point arithmetic distinguishes
        # between 0.0 and -0.0, which leads to incorrect sorting (see #3455)
```